### ISSUE TITLE
terraform crashes when creating VPC

### ISSUE THREAD

**Issue by yujunz** [alibaba/terraform-provider#417]
```
provider "alicloud" {
  region  = "cn-hongkong"
  version = "~> 1.8"
}

resource "alicloud_vpc" "default" {
  description = "default vswitch managed by terraform"
  cidr_block  = "172.31.0.0/16"
}
```

It works fine if I remove "description"

[crash.log](https://github.com/alibaba/terraform-provider/files/1821170/crash.log)

**Comment by xiaozhu36** [comment]
Hi @yujunz yeah, it is a bug. But now it has been fixed by [release 1.8.2](https://github.com/alibaba/terraform-provider/releases/tag/V1.8.2). Please update it.

**Comment by yujunz** [comment]
Fixed after updating 👍
